Vagrantfile

```
@@ -127,10 +128,12 @@ Vagrant.configure("2") do |config|
     # kubectl
     #
 
-    apt-get install --yes apt-transport-https ca-certificates curl
-    curl -fsSL https://packages.cloud.google.com/apt/doc/apt-key.gpg | gpg --dearmor -o /etc/apt/keyrings/kubernetes-archive-keyring.gpg
-    echo "deb [signed-by=/etc/apt/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | tee /etc/apt/sources.list.d/kubernetes.list
-    apt-get update
+    sudo apt-get install -y apt-transport-https ca-certificates curl
```

Using sudo generated again this undesired output: `default: dpkg-preconfigure: unable to re-open stdin: No such file or directory`. It's tackled by `export DEBIAN_FRONTEND=noninteractive` above. So to avoid it, we can continue running commands without sudo (we're already have full credentials).

```suggestion
    apt-get install -y apt-transport-https ca-certificates curl
```